network-protection/network-protection-impl/src/main/res/values/donottranslate.xml

```diff
@@ -269,11 +269,6 @@
     <string name="netpGeoswitchingHeaderCustom">Custom</string>
     <string name="netpGeoswitchingHeaderCountrySubtitle" instruction="Placeholder is a number greater than 1">%1$d cities</string>
 
```

Comment:
The whole `waitlist` and `terms` section here can be removed as well.

Reply:
All but two strings, will remove